lib/exgboost/parameters.ex

```diff
@@ -0,0 +1,968 @@
+defmodule EXGBoost.Parameters do
+  @global_params [
+    verbosity: [
+      type: {:custom, EXGBoost.Parameters, :validate_verbosity, []},
+      doc: """
+      Verbosity of printing messages. Valid values are:
+      `:silent`, `:warning`, `:info`, `:debug`.
+      """
+    ],
+    use_rmm: [
+      type: :boolean,
+      default: false,
+      doc: """
+      Whether to use RAPIDS Memory Manager for memory allocation.
```

Comment:
If this is only available in certain instances it should probably be a library config set globally and then grabbed from the application env.

Users would do:

```elixir
config :exgboost, use_rmm: true
```